src/test/java/com/weaverplatform/postgresql/database/TestQuery.java

```
@@ -12,21 +14,25 @@
  */
 public class TestQuery {
 
-  public static final String
+  public static String
     TEST_CREATE_NODE, TEST_REMOVED_NODE, TEST_LIST_DATABASES,
     TEST_NO_DATABASE, TEST_LIST_ATTRIBUTE, TEST_LIST_RELATION;
 
-  private static String read(String fileName) {
+  private static String read(String fileName) throws IOException {
     return Resource.get("queries/" + fileName);
   }
 
   static {
-    TEST_CREATE_NODE     = read("test_create_node.sql");
-    TEST_REMOVED_NODE    = read("test_removed_node.sql");
-    TEST_NO_DATABASE     = read("test_no_database.sql");
-    TEST_LIST_DATABASES  = read("test_list_database.sql");
-    TEST_LIST_ATTRIBUTE  = read("test_list_attribute.sql");
-    TEST_LIST_RELATION   = read("test_list_relation.sql");
+    try {
+      TEST_CREATE_NODE = read("test_create_node.sql");
+      TEST_REMOVED_NODE = read("test_removed_node.sql");
+      TEST_NO_DATABASE = read("test_no_database.sql");
+      TEST_LIST_DATABASES = read("test_list_database.sql");
+      TEST_LIST_ATTRIBUTE = read("test_list_attribute.sql");
+      TEST_LIST_RELATION = read("test_list_relation.sql");
+    } catch (IOException e) {
+      e.printStackTrace();
```

This should throw a runtime exception I guess, if any file cannot be found, testing should using these files should not continue

---

done